include/configs/imx8mm_var_dart.h

```diff
@@ -87,10 +87,12 @@
 		"bootm ${initrd_addr}#${fit_config};\0" \
 	"swu=1\0" \
 	"get_fit_config=leica_mcu_comm board_id board_id;" \
-		"if test \"${board_id}\" < \"D\"; then" \
-		"    env set fit_config conf-freescale_ap20-pt1.dtb;" \
-		"else" \
-		"    env set fit_config conf-freescale_ap20-pt1-5.dtb;" \
+		"if test \"${board_id}\" < \"D\"; then " \
+		    "echo !!! ERROR: Board with ID ${board_id} is not supported; " \
+		"elif test -n \"${downgrade_dtb}\"; then " \
+		    "env set fit_config conf-freescale_${downgrade_dtb}; " \
```

Comment:
I cannot seems to find `${downgrade_dtb}` variable. Where is it defined?

Reply:
HI @zandrey , the `${downgrade_dtb}` variable was defined in the geo_surv, please see the PR `https://github.hexagon.com/geo-surv/meta-geosurv/pull/332/files`: 
`        if [ "$hw" = "AP20-PT1.5" ] && [ "$hw_model" = "AP20" ];
        then
            hw_compatible="Y"
            bootenv_set downgrade_dtb "ap20-pt1-5.dtb"
            break
        fi
`